apps/condo/domains/property/constants.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Generated by `createservice property.ExportPropertiesToExcelService --type queries`
+ */
+
+module.exports = {
```

Comment:
Seems like this file is unnecessary or should we keep empty auto-generated files?

Reply:
Ok